src/components/VContentSwitcher/VContentSwitcherPopover.vue

```diff
@@ -9,6 +9,11 @@
       <VContentSwitcherButton
         :a11y-props="a11yProps"
         :active-item="activeItem"
+        :class="{
+          '!border-tx group-hover:!border-dark-charcoal-06':
```

Comment:
Great idea to use `!important`! I didn't know Tailwind can do it before seeing this. The [documentation](https://tailwindcss.com/docs/configuration#important) says it needs to be configured and I don't see that in our `tailwind.config.js` file so maybe with JIT mode it is not necessary?

Although the border is almost imperceptible, shouldn't that be the same value as the border of the input field? Sure @panchovm should know better.
```suggestion
          '!border-tx group-hover:!border-dark-charcoal-20':
```

Reply:
You are right @krysal, in `hover` the border should be `dark-charcoal-20`.

Reply:
And I was looking on the border trying to see why I can't really see it, @krysal ! Thank you so much for spotting this!